docs/conf.py

```diff
@@ -48,6 +48,7 @@
     "noaa": ("https://www.noaa.gov/%s", "%s"),
     "pylint": ("https://pylint.readthedocs.io/en/stable/%s", "%s"),
     "pytest": ("https://docs.pytest.org/en/7.4.x/%s", "%s"),
+    "regional-esg-grid": ("https://noaa-emcufs-utils.readthedocs.io/en/latest/ufs_utils.html#%s", "%s"),
```

Comment:
the driver itself is called `esg_grid`, but the documentation is referenced as `regional_esg_grid`.